requirements/base.txt

```diff
@@ -1,11 +1,13 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
Seems like `make upgrade` was ran with `Python 3.9`. We should run this with `Python 3.8` instead.